apps/aecore/lib/aecore/structures/oracle_extend_tx.ex

```diff
@@ -92,7 +94,7 @@ defmodule Aecore.Structures.OracleExtendTx do
       !Map.has_key?(registered_oracles, sender) ->
         throw({:error, "Account isn't a registered operator"})
 
-      fee < calculate_minimum_fee(tx.ttl) ->
+      fee < calculate_minimum_fee(tx.ttl) && fee < @minimum_fee ->
```

Comment:
There should be `||` here for correct behavior probably.

Reply:
gues this wouldn't be compatible if our fee is lower than epochs